ui/src/app/pages/signed-in/component.ts

```diff
@@ -128,9 +177,20 @@ export class SignedInComponent implements OnInit, OnDestroy, AfterViewInit {
   private navigateSignOut(): void {
     // Force a hard browser reload here. We want to ensure that no local state
     // is persisting across user sessions, as this can lead to subtle bugs.
-    window.location.assign('/');
+    window.location.assign('https://accounts.google.com/logout');
```

Comment:
Per slack, I would use this redirect approach to send them back to the AoU login page: https://stackoverflow.com/questions/4202161/google-account-logout-and-redirect